src/openrct2/peep/Guest.cpp

```diff
@@ -1748,7 +1748,7 @@ bool rct_peep::ShouldGoOnRide(int32_t rideIndex, int32_t entranceNum, bool atQue
             // Rides without queues can only have one peep waiting at a time.
             if (!atQueue)
             {
-                if (ride->last_peep_in_queue[entranceNum] != 0xFFFF)
+                if (ride->stations[entranceNum].LastPeepInQueue != 0xFFFF)
```

Comment:
0xFFFF should be replaced by the appropriate constant, while we're at it. (Also see a few lines below.)